fuel-vm/src/interpreter/debug.rs

```diff
@@ -119,6 +125,7 @@ fn breakpoint_script() {
 
 #[test]
 fn single_stepping() {
+    use alloc::vec;
```

Comment:
It seems you already should have this import from above.

Reply:
Removed in be98b794. Weird that `clippy` didn't catch this.